1-exercises/D-for-in-loops/exercise-1.js

```diff
@@ -13,6 +13,11 @@ let salaries = {
 };
 
 // WRITE BELOW THIS LINE
+for (let  name in salaries) {
```

Comment:
Great. just consider the variable names that you choose. for example what about 'price' instead of 'name'?